### ISSUE TITLE
Accessability Violation : The ARIA attributes "aria-label" are not valid for the element <div> with implicit ARIA role "generic"(ibm-code-snippet)

### ISSUE THREAD

**Issue by AlvinChacko** [carbon-design-system/carbon-components-angular#3085]
### Describe in detail the issue you're having ###

The ARIA attributes "aria-label" are not valid for the element with implicit ARIA role "generic"

### Is this a feature request (new component, new icon), a bug, or a general issue? ###

bug

### Is this issue related to a specific component? ###

ibm-code-snippet

### What did you expect to happen? What happened instead? What would you like to see changed? ###

Need to remove aria-label in div tag of ibm-code-snippet

### What browser are you working in? ###

Firefox

### What version of the Carbon Design System are you using? ###

"carbon-components-angular": "^5.54.0",

https://angular.carbondesignsystem.com/?path=/story/components-code-snippet--multi

Screenshot:
![Image](https://github.com/user-attachments/assets/72291ec0-967f-41e8-bd64-d9b73128285b)

### Additional Info ###
Looks like it was fixed for basic type before but not for multi:
https://github.com/carbon-design-system/carbon-components-angular/issues/2550
